packages/arb-token-bridge-ui/src/hooks/useArbQueryParams.tsx

```diff
@@ -97,6 +108,55 @@ export const AmountQueryParam = {
   }
 }
 
+// Parse chainId to ChainQueryParam or ChainId for orbit chain
+function encodeChainQueryParam(chainId: number | null | undefined) {
+  if (!chainId) {
+    return undefined
+  }
+
+  try {
+    const chain = getChainQueryParamForChain(chainId)
+    return chain.toString()
+  } catch (e) {
+    return undefined
+  }
+}
+
+function isValidNumber(value: number) {
+  return !Number.isNaN(value)
+}
```

Comment:
seems generic enough that it should be in a more common util file?

Reply:
I'm not sure about this one, I would move it once it's needed somewhere else. It's only here because the query param library might return null or undefined for a number, I'm not sure we have this case anywhere else.

Reply:
not a strong opinion but the name `isValidNumber` and what it does in the fn are totally reusable in any other functions when we want to check `isNaN`